trade-finance-manager-api/src/v1/api.js

```diff
@@ -302,6 +302,39 @@ const updateDealCancellation = async ({ dealId, dealCancellationUpdate, auditDet
   }
 };
 
+/**
+ * Submits the deal cancellation
+ * @param {Object} params
+ * @param {string} params.dealId - id of deal to cancel
+ * @param {import('@ukef/dtfs2-common').TfmDealCancellation} params.cancellation - the cancellation details to submit
+ * @param {import('@ukef/dtfs2-common').AuditDetails} params.auditDetails - user making the request
+ * @returns {Promise<import('@ukef/dtfs2-common').TfmDealCancellationResponse>} update result object
+ */
+const submitDealCancellation = async ({ dealId, cancellation, auditDetails }) => {
```

Comment:
Again same issue on submit vs post (submit is already being used in the API layer so can keep it as is and change dtfs central)